source/features/forked-to.tsx

```diff
@@ -30,8 +32,19 @@ const updateCache = cache.function(async (): Promise<string[] | undefined> => {
 	staleWhileRevalidate: 5
 });
 
+async function updateForkLinkToDefaultBranch(url: GitHubURL): Promise<void> {
+	if (await doesFileExist(url)) {
+		const defaultBranch = await getDefaultBranch(getRepositoryInfo(url.pathname.slice(1)));
+
+		select<HTMLAnchorElement>(`.pagehead-actions [href*="${url.pathname}"]`)!.pathname = url.assign({
+			route: 'edit',
```

Comment:
This is not restricted to `isEditingFile` at all. It applies to every type of link